client/lib/domain/auth/value_objects.dart

```diff
@@ -37,16 +37,11 @@ class Name extends ValueObject<String> {
 }
 
 class Role extends ValueObject<String> {
-  static const List<String> roles = [
```

Comment:
@BarokDG already implemeted it in #25 . Choose one

Reply:
yea but it kept saying assertion failed when i tried to run it

Reply:
you should prolly merge this and tell @BarokDG to delete his

Reply:
sure beka